records_mover/records/targets/table/move_from_fileobjs_source.py

```diff
@@ -30,7 +30,10 @@ def __init__(self,
         super().__init__(prep, target_table_details, processing_instructions)
 
     def load(self, driver: DBDriver) -> Optional[int]:
-        return driver.load_from_fileobj(schema=self.tbl.schema_name,
+        loader = driver.loader_from_fileobj()
+        # TODO: Should this take a loader?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker